xcdat/temporal.py

```diff
@@ -695,12 +695,9 @@ def center_times(self, dataset: xr.Dataset) -> xr.Dataset:
         xr.Dataset
             The Dataset with centered time coordinates.
         """
-        ds = dataset.copy()
-
-        if hasattr(self, "_time_bounds") is False:
-            self._time_bounds = ds.bounds.get_bounds("time")
+        ds = self._dataset.copy()
+        time_bounds = ds.bounds.get_bounds("time")
```

Comment:
This update to `center_times()` makes it callable as a standalone method, instead of having to pass a dataset object to it. (from `ds.temporal.center_times(ds)` to `ds.temporal.center_times()`)